Add OKR process

dcwalk opened the pull request.

benhylau commented:
How come all these internal link errors. How did the previous PRs succeed?
```
- ./_book/guides.html
  *  internally linking to l#using-your-existing-email-client, which does not exist (line 0)
     <a href="l#using-your-existing-email-client">If you'd like to use it from another email client...</a>
  *  linking to internal hash #using-Gmail that does not exist (line 0)
     <a href="#using-Gmail">If you'd like to use it from your existing Gmail account...</a>
htmlproofer 3.15.2 | Error:  HTML-Proofer found 5 failures!
  *  linking to internal hash #using-the-web-interface that does not exist (line 0)
     <a href="#using-the-web-interface">If you'd like to use it from our hosted webmail interface...</a>
  *  linking to internal hash #using-your-existing-email-client that does not exist (line 0)
     <a href="l#using-your-existing-email-client">If you'd like to use it from another email client...</a>
- ./_book/reading-pool.html
  *  External link https://www.willystreet.coop/pages/seven-cooperative-principles failed: got a time out (response code 0)
##[error]Process completed with exit code 1.
```

dcwalk commented:
> How come all these internal link errors. How did the previous PRs succeed?
> 
> ```
> - ./_book/guides.html
>   *  internally linking to l#using-your-existing-email-client, which does not exist (line 0)
>      <a href="l#using-your-existing-email-client">If you'd like to use it from another email client...</a>
>   *  linking to internal hash #using-Gmail that does not exist (line 0)
>      <a href="#using-Gmail">If you'd like to use it from your existing Gmail account...</a>
> htmlproofer 3.15.2 | Error:  HTML-Proofer found 5 failures!
>   *  linking to internal hash #using-the-web-interface that does not exist (line 0)
>      <a href="#using-the-web-interface">If you'd like to use it from our hosted webmail interface...</a>
>   *  linking to internal hash #using-your-existing-email-client that does not exist (line 0)
>      <a href="l#using-your-existing-email-client">If you'd like to use it from another email client...</a>
> - ./_book/reading-pool.html
>   *  External link https://www.willystreet.coop/pages/seven-cooperative-principles failed: got a time out (response code 0)
> ##[error]Process completed with exit code 1.
> ```

I'm not sure. I thought I might have made a mistake on not starting from current head, but that isn't it. I _know_ we've fixed that coop principles one before. Did some changes get reverted in a recent PR?

dcwalk commented:
@benhylau can I get a re-review? updated copy after our meeting. Fixed the errors-- one was introduced in last PR (not sure why it wasn't caught), other maybe a regression...? Anyway, fixed on master, merged in here.
